src/imagery/i.eodag/i.eodag.py

```diff
@@ -879,11 +879,7 @@ def skip_existing(output, search_result):
             if scene_file.exists():
                 creation_time = datetime.utcfromtimestamp(os.path.getctime(scene_file))
                 ingestion_time = scene.properties.get("modificationDate")
```

Comment:
```suggestion
                ingestion_time = scene.properties.get(
                    "modificationDate",
                    scene.properties.get(
                        "publicationDate", scene.properties.get("creationDate")
                    ),
                )
```

Rollback in case modificationDate doesn't exist...